docs/content/sre/tenant-operator/customresources.md

```diff
@@ -227,11 +227,24 @@ resources:
           ports:
           - protocol: TCP
             port: 5978
+---
+apiVersion: tenantoperator.stakater.com/v1alpha1
+kind: Template
+metadata:
+  name: resource-mapping
+resources:
+  resourceMappings:
+    secrets:
+      - name: secret-s1
+        namespace: namespace-n1
+    configMaps:
+      - name: configmap-c1
+        namespace: namespace-n2
 ```
 
-Templates are used to initialize Namespaces and share common resources across namespaces (e.g. secrets).
+Templates are used to initialize Namespaces, share common resources across namespaces, and map secrets and configMaps from one namespace to other namespaces.
```

Comment:
'and map secret/configMaps from one namespace to other namespaces.'
^ Update it to this if this makes more sense.